src/main/java/dbdr/security/service/LoginService.java

```diff
@@ -0,0 +1,46 @@
+package dbdr.security.service;
+
+import dbdr.domain.careworker.dto.request.CareworkerRequestDTO;
+import dbdr.domain.guardian.dto.request.GuardianRequest;
+import dbdr.security.Role;
+import dbdr.security.dto.BaseUserDetails;
+import dbdr.security.dto.LoginRequest;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Slf4j
+public class LoginService {
+
+    private final AuthenticationManagerBuilder authenticationManagerBuilder;
+
+    private final JwtProvider jwtProvider;
+
+    private final Long jwtExpiration;
```

Comment:
```suggestion
    @Value("${spring.jwt.jwtexpiration}")
    private final Long jwtExpiration;
```
이렇게 사용하는 것은 어떤가요? 개인적인 생각입니다.

Reply:
final 사용을 위해서 그렇게 하였는데, 오히려 시인성측면에서는 위 방법이 좋은 것 같네용!!